src/solvers/jesse_dg/elixir_esdg_compressible_Euler_1d.jl

```diff
@@ -0,0 +1,79 @@
+using OrdinaryDiffEq, Trixi
+
+using StructArrays
+using RecursiveArrayTools,ArrayInterface
+using LazyArrays
+
+using LinearAlgebra,BenchmarkTools,Plots,UnPack
+
+using Octavian,CheapThreads
+using StartUpDG
+
+include("trixi_interface.jl") 
+include("flux_differencing.jl")
+include("ModalESDG.jl")
+include("modal_esdg.jl")
+
+###############################################################################
+# semidiscretization 
+
+N = 3
+K1D = 32
+CFL = .1
+FinalTime = .7
+
+rd = RefElemData(Line(),N)
+VX,EToV = uniform_mesh(Line(),K1D)
+md = MeshData(VX,EToV,rd)
+md = make_periodic(md,rd)
+
+eqn = CompressibleEulerEquations1D(1.4)
+function LxF_dissipation(u_ll, u_rr, orientation, equations::CompressibleEulerEquations1D)
+    return .5 * max_abs_speed_naive(u_ll, u_rr, orientation, equations) * (u_ll-u_rr)
+end
+solver = ModalESDG(rd,Trixi.flux_chandrashekar,Trixi.flux_chandrashekar,LxF_dissipation,
+                   Trixi.cons2entropy,Trixi.entropy2cons,eqn)
+
+function initial_condition(xyz,t,equations::CompressibleEulerEquations1D)
+    x, = xyz
+    ρ = 1 + .98*sin(pi*x)
+    u = 1.0
+    p = 2.
+    return prim2cons((ρ,u,p),equations)
```

Comment:
This is just a minor aspect and probably bikeshedding... Would you mind using `\varrho` instead of `\rho` for the density? I find it sometimes quite hard to distinguish `ρ` and `p`, in particular on GitHub. `ϱ` is much easier for me to parse for many fonts. I'm not set on this, it's just a suggestion.

Reply:
I remember the conversation on unicode - I'm happy to change to `\varrho` (or just sticking with `rho`).